doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -2006,232 +2158,360 @@ statistics will be shown, based on the sorting.
-      filename: packet_stats.log                    #name of the file in which packet profiling information will be
-                                                    #stored.
-      append: yes                                   #If set to yes, new packet profiling information will be added to the
-                                                    #information that was saved last in the file.
+Napatech
+~~~~~~~~
 
-      # per packet csv output
-      csv:
+.. TODO napatech doc
 
-        # Output can be disabled here, but it will still have a
-        # performance impact if compiled in.
+Rules
+-----
 
-        enabled: no                                #the sending of packet output to a csv-file is by default disabled.
-        filename: packet_stats.csv                 #name of the file in which csv packet profiling information will be
-                                                   #stored
+Rule-files
+~~~~~~~~~~
 
-Packet profiling is enabled by default in suricata.yaml but it will
-only do its job if you compiled Suricata with --enable profiling.
+For different categories of risk there are different rule-files
+available containing one or more rules. There is a possibility to
+instruct Suricata where to find these rules and which rules you want
+to be load for use. You can set the directory where the files can be
+found.
 
-The filename in which packet profiling information will be stored, is
-packet-stats.log. Information in this file can be added to the last
-information that was saved there, or if the append option is set to
-no, the existing file will be overwritten.
+::
 
-Per packet, you can send the output to a csv-file. This file contains
-one line for each packet with all profiling information of that
-packet. This option can be used only if Suricata is build
-with --enable-profiling and if the packet profiling option is enabled
-in yaml.
+  default-rule-path: /etc/suricata/rules/
+  rule-files:
+    - backdoor.rules
+    - bad-traffic.rules
+    - chat.rules
+    - ddos.rules
+    - ....
 
-It is best to use runmode 'single' if you would like to profile the
-speed of the code.  When using a single thread, there is no situation
-in which two threads have to wait for each other. When using two
-threads, the time threads might have to wait for each other will be
-taken in account when/during profiling packets. For more information
-see :doc:`../performance/packet-profiling`.
+The above mentioned is an example of rule-files of which can be chosen
+from. There are much more rule-files available.
 
-Application layers
-------------------
+If wanted, you can set a full path for a specific rule or
+rule-file. In that case, the above directory (/etc/suricata/rules/)
+will be ignored for that specific file. This is convenient in case you
+write your own rules and want to store them separate from other rules
+like that of VRT, ET or ET pro.
 
-SSL/TLS
-~~~~~~~
+If you set a file-name that appears to be not existing, Suricata will
```

Comment:
Rule files that don't exist are ignored by Suricata (a error (warning?)) message will be displayed).